system/jlib/jsocket.cpp

```diff
@@ -3313,82 +3415,55 @@ static bool decodeNumericIP(const char *text,unsigned *netaddr)
     return false;
 }
 
-static bool lookupHostAddress(const char *name,unsigned *netaddr)
+static void RecursionSafeLogErr(int ret, int ref, const char *msg, unsigned lineno, const char *name)
 {
-    // if IP4only or using MS V6 can only resolve IPv4 using 
     static bool recursioncheck = false; // needed to stop error message recursing
```

Comment:
Not new/to fix in this PR, but this should probably be a thread local variable.

Reply:
changed.